lms/db/__init__.py

```diff
@@ -160,10 +154,9 @@ def includeme(config):
     # Create the SQLAlchemy engine and save a reference in the app registry.
     engine = make_engine(config.registry.settings)
     config.registry["sqlalchemy.engine"] = engine
-    init(engine)
```

Comment:
This is the main change, don't just init right away, this will create the tables that alembic is expected to create on the second step of `make db` because if alebmic_vesion exist we are not stamping so we were trying to do both, `create_all` and `alembic upgrade head` (without stamping)